packages/pyright-internal/src/localization/localize.ts

```diff
@@ -53,12 +53,16 @@ type MessageStyle = 'default' | 'simplified';
 
 let messageStyle: MessageStyle = 'default';
 
+export function isSimpleMessageStyle() {
+    return messageStyle === "simplified";
+}
+
 export function setMessageStyle(style: MessageStyle) {
     messageStyle = style;
 }
 
 export function optionalAddendum(diag: DiagnosticAddendum) {
-    return messageStyle === 'simplified' ? '' : diag.toString();
+    return isSimpleMessageStyle() ? '' : diag.getString();
```

Comment:
The switch from toString -> getString fixes an issue introduced in #11.
We should retain this even if we don't merge this PR.